README.md

```diff
@@ -79,23 +79,22 @@ repo contains the `pulumi` CLI, language SDKs, and core Pulumi engine, and indiv
 
 <img align="right" width="400" src="https://www.pulumi.com/images/docs/quickstart/console.png" />
 
-* **[Getting Started](#getting-started)**: get up and running quickly.
+* **[Getting Started](#getting-started)**: Deploy a simple application in AWS, Azure, Google Cloud, or Kubernetes using Pulumi.
```

Comment:
this doesnt actually link to the getting started guides, but it is linked to in the section this links to. let me know if yall think this description is confusing knowing that.

Reply:
I think it's a little confusing to have an internal link, followed by a bunch of external links. Especially since it links to the section right below this one. Maybe change it to
```suggestion
* **[Getting Started](https://www.pulumi.com/docs/get-started/)**: Deploy a simple application in AWS, Azure, Google Cloud, or Kubernetes using Pulumi.
```
You could also remove the bullet all together, since people will notice the `## Getting Started` tag just below.